helpers/parse_args.go

```diff
@@ -12,18 +12,23 @@ import (
 	"os"
 	"strings"
 
+	"github.com/Azure/go-autorest/autorest/azure"
 	"github.com/subosito/gotenv"
 )
 
+const (
+	publicCloudName = "AZUREPUBLICCLOUD"
```

Comment:
Is this const something that should be defined in go-autorest?

Reply:
It sort of is already: https://github.com/Azure/go-autorest/blob/9ad9326b278af8fa5cc67c30c0ce9a58cc0862b2/autorest/azure/environments.go#L63